stats.cpp

```diff
@@ -1,5 +1,55 @@
 #include "stats.h"
+#include<cmath>
+using namespace std;
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Statistics::Stats Statistics::ComputeStatistics(const std::vector<float>& a) {
     //Implement statistics here
+     Statistics::Stats s;
+    if(a.size()==0)
+    {
+         s.average=nanf("");
+         s.max=nanf("");
+         s.min=nanf("");
```

Comment:
Could these initializations be in the constructor of the Stats structure? That way we don't need to worry about un-initialized variables in our program.